### DISCUSSION TITLE
[Solved] Problem with rendering romanization of the Japanese author's name in citation key

### DISCUSSION THREAD

**Discussion by DarrenMMcDonald** [retorquere/zotero-better-bibtex#2652] [Q&A] [answered]
This is an issue I originally posted in the main Zotero forum which I am also posting here for other users as **the solution was found**.

This was the **original issue**:
I am having a problem rendering the romanization of Japanese (kanji) names in the citation key. Currently, Zotero, via the Better Bibtex plugin, is rendering the author's name into the romanization of Chinese, rather than the Japanese.

For example for "一小路武安2016日本における、、、" is being rendered as:
YiXiaoLu2016nihonniokeru...

This should be rendered as:
ichokoji2016nihonnniokeru...

Note this issue only affects the author's name, not the title.
In the Better Bibtex plugin preferences, I have checked the "Apply kuroshiro romanization in Japanese names/titles" under "Ideographs in citekeys".

**The solution:**
Make sure that the "Language" is filled in correctly in info>Language>[input "日本語” or "ja"]. 
This info window appears for each item that you add to Zotero. In my case, it appears on the right-hand side of the main Zotero window when you select an item.

Also, in the Better Bibtex plugin preferences, check the "Apply kuroshiro romanization in Japanese names/titles" under "Ideographs in citekeys".
kuroshiro romajization option (in Better BibTex preferences).

In this case, the issue was entirely my fault for not filling the "language" in the "info" window of the item. I now realize why I need to fill in this for each item. ;)

**Comment by retorquere** [comment, marked as answer]
The solution:
Make sure that the "Language" is filled in correctly in info>Language>[input "日本語” or "ja"].
This info window appears for each item that you add to Zotero. In my case, it appears on the right-hand side of the main Zotero window when you select an item.

Also, in the Better Bibtex plugin preferences, check the "Apply kuroshiro romanization in Japanese names/titles" under "Ideographs in citekeys".
kuroshiro romajization option (in Better BibTex preferences).
